corehq/apps/hqmedia/templates/hqmedia/bulk_upload.html

```diff
@@ -1,4 +1,4 @@
-{% extends "hqmedia/uploader_base.html" %}
```

Comment:
Can't you delete `hqmedia/uploader_base.html` altogether? I only see two references to it, and you're updating both of them.